### DISCUSSION TITLE
Unable to Delete Users

### DISCUSSION THREAD

**Discussion by jesus-hergueta** [mozilla/hubs#3302] [Hubs Cloud Questions] [answered]
Hi.

I have a problem with my Hubs instance.

I am running an instance of Mozilla Hubs Enterprise Multi-Server on AWS. And everything is going perfectly but I can't delete my users. I can create and modify them (enable / disable, mark them as admin or not, even add them to identities) but when I select them to delete them the page refreshes, shows an error saying "URL not found" and then redirects me to the main Hubs page ( outside the admin dashboard).

When I go back in the user list remains the same.

Thanks.

**Comment by robinkwilson** [comment, marked as answer]
Thanks for the note! This is a duplicate of: https://github.com/mozilla/hubs/discussions/2747

More related discussions:
https://github.com/mozilla/hubs/discussions/3142
